Firestore/Example/Benchmarks/remote_document_cache_benchmark.mm

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2020 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <FirebaseCore/FIRApp.h>
+#import <FirebaseFirestore/FirebaseFirestore.h>
+
+#include "Firestore/core/src/firebase/firestore/util/autoid.h"
+#include "Firestore/core/src/firebase/firestore/util/hard_assert.h"
+#include "Firestore/core/src/firebase/firestore/util/string_apple.h"
+#include "Firestore/core/test/firebase/firestore/testutil/app_testing.h"
+#include "benchmark/benchmark.h"
+
+namespace {
+
+using firebase::firestore::testutil::AppForUnitTesting;
+using firebase::firestore::util::CreateAutoId;
+using firebase::firestore::util::MakeNSString;
+using firebase::firestore::util::MakeString;
+
+FIRFirestore* OpenFirestore() {
+  FIRApp* app = AppForUnitTesting();
+  auto db = [FIRFirestore firestoreForApp:app];
+  auto settings = db.settings;
+  settings.host = @"localhost:8080";
+  settings.sslEnabled = NO;
+  settings.dispatchQueue = dispatch_queue_create("results", DISPATCH_QUEUE_SERIAL);
+  db.settings = settings;
+
+  return db;
+}
+
+NSMutableDictionary<NSString*, id>* MakeDocument() {
```

Comment:
This is more of a MakeData() function and looks like it could potentially just be replaced with a single constant that initializes a dictionary.